libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -1,4 +1,4 @@
-<form [formGroup]="searchForm" (submit)="searchBooks()">
+<form [formGroup]="searchForm" (submit)="searchForBooks()">
```

Comment:
What's the reasoning for this refactor?

Reply:
Because the method searchBooks() had exactly the same name as the searchBooks imported on the component.ts file, so it was kind of confusing, so I thought it was best to change a bit the name, leaving still clear what it does.